notebooks/en/rag_with_unstructured_data.ipynb

```diff
@@ -0,0 +1,509 @@
+{
```

Comment:
<p>I think this paragraph would make more sense in the Chunking section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>